bitnami/postgresql-ha/values.yaml

```diff
@@ -154,6 +154,9 @@ postgresql:
   ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
   ##
   updateStrategyType: RollingUpdate
+  ## @param postgresql.containerPort PostgreSQL port
+  ##
+  containerPort: 5432
```

Comment:
I'm missing the equivalent (`pgpool.containerPort`) for Pgpool, right?

Reply:
I just added it to the values, README and pgpool deployment